src/ast/passes/parser.h

```diff
@@ -59,6 +59,7 @@ inline std::vector<Pass> AllParsePasses(
   passes.emplace_back(CreateMapSugarPass());
   passes.emplace_back(CreateNamedParamsPass());
   passes.emplace_back(CreatePidFilterPass());
+  passes.emplace_back(CreateResolveImportsPass());
```

Comment:
`ResolveImports`and `ResolveRootImports` are overlapping and confusing. Let's ensure they are disjoint. E.g. `ResolveStatementImports` and `ResolveRootImports`?